src/services/webdav/backend.rs

```diff
@@ -99,6 +99,8 @@ impl Debug for WebdavBuilder {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
         let mut de = f.debug_struct("Builder");
         de.field("endpoint", &self.endpoint);
+        de.field("username", &self.username);
```

Comment:
**request (blocking)**: Please don't print username and password in Debug. User could print them in logging unintended.